Makefile.eve

```diff
@@ -1,12 +1,7 @@
 # Title: Dockerfile for building the EVE kernel
 VERSION=$(shell git rev-parse --short HEAD)
-DIRTY=$(shell git diff --quiet || echo '-dirty')
-EVE_FLAVOR=generic
 ARCHITECTURE=amd64
-KERNEL_TAG=$(shell git describe --tags --abbrev=0)
-BRANCH=eve-kernel-$(ARCHITECTURE)-$(KERNEL_TAG)-$(EVE_FLAVOR)
```

Comment:
Please do BRANCH?=eve-kernel-$(ARCHITECTUE)-v6.1.38-$(EVE_FLAVOR)
When makefile is not called from GH action we need BRANCH to be set

Reply:
Added `BRANCH=eve-kernel-$(ARCHITECTURE)-v6.1.38-$(EVE_FLAVOR)` in the Makefile.eve for the local execution.   Updated the run command in the GH action to pass the `BRANCH` which will take precedence.